habitat-lab/habitat/core/benchmark.py

```diff
@@ -24,14 +24,21 @@ class Benchmark:
     r"""Benchmark for evaluating agents in environments."""
 
     def __init__(
-        self, config_paths: Optional[str] = None, eval_remote: bool = False
+        self,
+        config_paths: Optional[str] = None,
+        eval_remote: bool = False,
+        action_space: Optional[str] = None,
```

Comment:
Yes, I know this is a hack but I specifically didnt want to create a list of overrides because that would allow the challenge participants to pass any number of config changes to the simulator which potentially opens the gate of the challenge to cheating.

Reply:
I am open to discussions about this though.

Reply:
Yes, we actually shouldn't promote user do overwrites this way.